### PR TITLE
[blockly] Add tooltips to the show block/code buttons

### PR DESCRIPTION
**Pull request by jimtng**

### CODE REVIEW

File: bundles/org.openhab.ui/web/src/pages/settings/rules/script/script-edit.vue [hunk 1]
```diff
@@ -27,8 +27,8 @@
       <span class="display-flex flex-direction-row align-items-center">
         <f7-button v-if="!newScript && isBlockly && !blocklyCodePreview" outline small :active="blocklyShowLabels" icon-f7="square_on_circle" :icon-size="($theme.aurora) ? 20 : 22" class="no-ripple" style="margin-right: 5px" @click="toggleBlocklyItemLabelId" :tooltip="'Toggle to show either Item labels or IDs'" />
         <f7-segmented v-if="!newScript && isBlockly" class="margin-right">
-          <f7-button outline small :active="!blocklyCodePreview" icon-f7="ticket" :icon-size="($theme.aurora) ? 20 : 22" class="no-ripple" @click="blocklyCodePreview = false" />
-          <f7-button outline small :active="blocklyCodePreview" icon-f7="doc_text" :icon-size="($theme.aurora) ? 20 : 22" class="no-ripple" @click="showBlocklyCode" />
+          <f7-button outline small :active="!blocklyCodePreview" icon-f7="ticket" :icon-size="($theme.aurora) ? 20 : 22" class="no-ripple" @click="blocklyCodePreview = false" :tooltip="'Show blocks'" />
```
Comment (florian-h05) [inline bundles/org.openhab.ui/web/src/pages/settings/rules/script/script-edit.vue:30]:
```suggestion
          <f7-button outline small :active="!blocklyCodePreview" icon-f7="ticket" :icon-size="($theme.aurora) ? 20 : 22" class="no-ripple" @click="blocklyCodePreview = false" tooltip="Show blocks" />
```

File: bundles/org.openhab.ui/web/src/pages/settings/rules/script/script-edit.vue [hunk 2]
```diff
@@ -27,8 +27,8 @@
...
+          <f7-button outline small :active="blocklyCodePreview" icon-f7="doc_text" :icon-size="($theme.aurora) ? 20 : 22" class="no-ripple" @click="showBlocklyCode" :tooltip="'Show generated code'" />
```
Comment (florian-h05) [inline bundles/org.openhab.ui/web/src/pages/settings/rules/script/script-edit.vue:31]:
```suggestion
          <f7-button outline small :active="blocklyCodePreview" icon-f7="doc_text" :icon-size="($theme.aurora) ? 20 : 22" class="no-ripple" @click="showBlocklyCode" tooltip="Show generated code" />
```

**Review by florian-h05 — COMMENTED** [review]
Thanks, I’ve just left two comments to simplify the code.

File: bundles/org.openhab.ui/web/src/pages/settings/rules/script/script-edit.vue (hunk 2 above)
Comment (jimtng) [reply bundles/org.openhab.ui/web/src/pages/settings/rules/script/script-edit.vue]: Fixed. Also fixed the other block from which I copied these :)

**Review by florian-h05 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**